lib/galaxy/managers/short_term_storage.py

```diff
@@ -24,13 +24,13 @@
     ObjectNotFound,
 )
 from galaxy.exceptions.error_codes import error_codes_by_int_code
+from galaxy.exceptions.utils import api_error_to_dict
 from galaxy.schema.schema import OptionalNumberT
 from galaxy.util import (
     directory_hash_id,
     is_uuid,
     safe_makedirs,
 )
-from galaxy.web.framework.decorators import api_error_message
```

Comment:
Could STS be placed in its own package now with this change? All it depends on now is galaxy-util and galaxy schema stuff (and then only for the typing).

Reply:
Yes sure! Do you mean a whole new package? or do you have an idea where to put it?

Reply:
galaxy.microservices? :thinking:

Reply:
I think the idea of a microservices package runs counter the idea of doing microservices 😅. I think the path should maybe just be ``galaxy.sts`` or ``galaxy.short_term_storage``. As to what Python package that module belongs to... that is trickier. 

I would think that should be in its own package based on its dependencies. If that is too heavy moving it to galaxy.web_stack would be possible after this commit since you've refactored the galaxy.web.framework pieces out. It doesn't belong there conceptually but it is a lighter weight import than all the web framework stuff. You could also keep it in web framework for now and just add a TODO somewhere that when we manage to decompose tools from the web framework we need to spin STS out.

Reply:
> I think the idea of a microservices package runs counter the idea of doing microservices 😅

Hah true! I see your point now :laughing:

Ok, I will see what I can do and in the worst case we do the TODO as suggested :+1: